apps/ui/src/style.scss

```diff
@@ -440,3 +442,203 @@ input:placeholder-shown {
 div[data-tippy-root] {
   @apply break-words;
 }
+
+// Markdown rendered content
+
+.markdown-body {
+  @apply text-skin-content text-[22px] leading-[1.3];
+  word-wrap: break-word;
+
+  > *:first-child {
+    @apply mt-0 #{!important};
+  }
+
+  > *:last-child {
+    @apply mb-0 #{!important};
+    @apply pb-0 #{!important};
+  }
+
+  a {
+    @apply underline;
+
+    &:not([href]) {
+      @apply no-underline text-inherit;
+    }
+  }
+
+  img {
+    &[alt^='discourse-emoji'] {
+      @apply inline size-[20px];
+    }
+
+    &[alt^='discourse-thumbnail'] {
+      @apply inline size-[35px];
+    }
+  }
+
+  p,
+  blockquote,
+  ul,
+  ol,
+  dl,
+  table,
+  pre {
+    @apply mt-0 mb-3;
+  }
+
+  p {
+    @apply text-[1em];
+  }
+
+  hr {
+    @apply h-[0.25em] p-0 my-4 mx-0 bg-[#e1e4e8] border-0;
+  }
+
+  blockquote {
+    @apply py-0 px-[1em] text-skin-text border-l-[0.25em] border-skin-text;
+
+    > :last-child {
+      @apply mb-0;
+    }
+
+    > :first-child {
+      @apply mt-0;
+    }
+  }
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    @apply mb-3 mt-4 font-semibold;
+    @apply leading-[1.4] #{!important};
+  }
+
+  h1,
+  h2 {
+    @apply text-[1.25em];
+  }
+
+  h3,
+  h4,
+  h5,
+  h6 {
+    @apply text-[1em];
+  }
+
+  ul,
+  ol {
+    @apply pb-2 pl-[1em];
+
+    &.no-list {
+      @apply p-0 list-none;
+    }
+
+    li {
+      word-wrap: break-all;
+
+      & + li {
+        @apply mt-[0.25em];
+      }
+    }
+
+    ul,
+    ol {
+      @apply mt-0 mb-0 pb-0;
+
+      li {
+        @apply mt-[0.25em];
+      }
+    }
+  }
+
+  ul {
+    @apply list-disc;
+  }
+
+  ol {
+    @apply list-decimal;
+  }
+
+  code {
+    @apply bg-skin-border text-sm rounded-md px-1.5 py-0.5;
+  }
+
+  dl {
+    @apply p-0;
+
+    dt {
+      @apply p-0 mt-3 font-semibold italic text-[1em];
+    }
+
+    dd {
+      @apply py-0 px-3 mb-3;
+    }
+  }
+
+  table {
+    @apply block w-full overflow-auto;
+
+    th {
+      @apply font-semibold;
+    }
+
+    th,
+    td {
+      @apply py-1.5 px-[13px] border border-skin-border;
+    }
+
+    tbody tr {
+      @apply bg-skin-bg;
+    }
+
+    thead tr,
+    tbody tr:nth-child(2n) {
+      @apply bg-skin-bg border border-[#c6cbd1];
+    }
+
+    img {
+      @apply bg-transparent;
+    }
+  }
+
+  // Mute paragraphs style inside table cells and list items
+  // All list should be styled as markdown tight list
+  li,
+  td,
+  th {
+    > p {
+      @apply m-0 p-0;
+    }
+  }
+
+  @include meta.load-css('@/assets/styles/highlightjs/github.scss');
```

Comment:
On terminal it show this notice 
```
Deprecation Warning [import]: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.
More info: https://sass-lang.com/d/import
```
didn;t check if it happen on master

Reply:
this deprecation warning should have been fixed with https://github.com/snapshot-labs/sx-monorepo/pull/1580/commits/fe78dbccf44a8b3fdf27656ba8f42e58eecc0b2b , your branch is not up to date